### DISCUSSION TITLE
How to Partially Observable

### DISCUSSION THREAD

**Discussion by jia-xie** [isaac-sim/IsaacLab#1473] [Q&A]
`mdp.joint_pos_rel` is one of the most common observations.
```python
def joint_pos_rel(env: ManagerBasedEnv, asset_cfg: SceneEntityCfg = SceneEntityCfg("robot")) -> torch.Tensor:
    """The joint positions of the asset w.r.t. the default joint positions.

    Note: Only the joints configured in :attr:`asset_cfg.joint_ids` will have their positions returned.
    """
    # extract the used quantities (to enable type-hinting)
    asset: Articulation = env.scene[asset_cfg.name]
    return asset.data.joint_pos[:, asset_cfg.joint_ids] - asset.data.default_joint_pos[:, asset_cfg.joint_ids]
```

And when initialing Observation Term, `mdp.joint_pos_rel` is passed in as `func` that is later called to retrieve the observation tensor.
```python
joint_pos = ObsTerm(func=mdp.joint_pos_rel, noise=Unoise(n_min=-0.01, n_max=0.01))
```
### Question

If i have a robot with a joint that is passive and does not have sensor feedback, how can I make `mdp.joint_pos_rel` only return part of the joint position instead of all the joints?

**Comment by GiulioRomualdi** [comment]
Hi @jia-xie . You can pass the joints list in the asset_cfg
